vllm/lora/utils.py

```diff
@@ -91,28 +92,44 @@ def replace_submodule(model: nn.Module, module_name: str,
     return new_module
 
 
-def parse_fine_tuned_lora_name(name: str) -> Tuple[str, bool, bool]:
+def parse_fine_tuned_lora_name(
+        name: str,
+        weights_mapper: Optional[WeightsMapper] = None
+) -> Tuple[str, bool, bool]:
     """Parse the name of lora weights.
 
     args:
         name: the name of the fine-tuned LoRA, e.g.
             base_model.model.dense1.weight
+        weights_mapper: maps the name of weight, e.g.
+            `model.` -> `language_model.model.`,
     return:
         Tuple(module_name, is_lora_a):
             module_name: the name of the module, e.g. model.dense1,
             is_lora_a whether the tensor is lora_a or lora_b.
             is_bias whether the tensor is lora bias.
     """
+    # TODO: Currently only supports mapping for prefix, mapping for substr and
+    # subfix will be supported in the future.
+    if weights_mapper is not None:
+        weights_mapper.orig_to_new_substr = {}
+        weights_mapper.orig_to_new_suffix = {}
```

Comment:
From my understanding, this will overwrite the original weights mapper. It may be safer to modify a copy of the original one.

Also, can you print a warning for unsupported mappings?

Reply:
Make sense

Reply:
I think a warning would be more appropriate?